tests/testthat/test_jobs_to_df.R

```diff
@@ -0,0 +1,35 @@
+# Remember, on Mac, this context line is REQUIRED. This is possibly a bug with testthat version 2.0.0.
+context("Test jobs_to_df and job_to_df functions")
+
+# This doesn't work -- why not? R Scope issues, I presume?
+# testthat::setup({
+#   jobs <- dget(file = system.file("tests/testthat/job_list_four_items.txt", package = "rrsq"))
+# })
+
+
+test_that(desc = "Dataframe is returned from job_to_df (singular)", {
+  jobs <- dget(file = system.file("tests/testthat/job_list_four_items.txt", package = "rrsq"))
+  job = jobs[[2]]
+  tryCatch({
+    result <- job_to_df(job)
+  }, error = function(e){
+    testthat::fail(message = paste0("Error occured during test: ", e$message))
```

Comment:
While I was developing the tests I was getting an error (due to the camel case being returned by RSQ), and I thought at the time that the error was messing up the test output. Turns out the real issue was that I didn't have a context(...) statement at the top. I haven't revisited whether or not the tryCatch is necessary since, but it was helpful having the test print the error message.